gateway-js/src/config.ts

```diff
@@ -126,6 +126,13 @@ interface GatewayConfigBase {
   experimental_autoFragmentization?: boolean;
   fetcher?: typeof fetch;
   serviceHealthCheck?: boolean;
+  /**
+   * @deprecated This configuration option shouldn't be used unless by
+   *             recommendation from Apollo staff. This behavior will be
+   *             defaulted in a future release and this option will strictly be
+   *             used as an override.
+   */
+  experimental_schemaConfigDeliveryEndpoint?: null;
```

Comment:
(we chatted on Slack about improving the typings here)

Reply:
f90cdfc